cgi-bin/ljlib.pl

```diff
@@ -1004,34 +1004,32 @@ sub start_request
             if LJ::is_enabled('esn_ajax');
 
         # contextual popup JS
-        if ( $LJ::CTX_POPUP ) {
```

Comment:
Yes, I meant to do this. I'm removing things that are dead weight, particularly tunables that we don't really want to support. We don't really have many sites using DW code (if any?) so we shouldn't promulgate this arcane knowledge IMO.